### ISSUE TITLE
Generated plugin doesn't work with stable Kibana version

### ISSUE THREAD

**Issue by xSAVIKx** [elastic/generator-kibana-plugin#40]
Hi, I've just cloned Kibana 5.0 branch and wanted to use plugin, generated with this generator, but it won't start with current kibana version, as it's 5.0.1 in 5.0 branch and plugin-helper wants 5.0.0:
```
npm start

> test_plugin@5.0.0 start /home/isergiichuk/Projects/test-plugin
> plugin-helpers start

 watching for changes  (298 files)
  log   [14:40:17.585] [info][listening] basePath Proxy running at https://localhost:5601/uol
server    log   [14:40:19.800] [warning] Plugin "test_plugin" was disabled because it expected Kibana version "5.0.0", and found "5.0.1".
optmzr    log   [14:40:19.837] [warning] Plugin "test_plugin" was disabled because it expected Kibana version "5.0.0", and found "5.0.1".

```

After changing version in `package.json` from `5.0.0` to `5.0.1` everything will work again.

Probably, this should somehow be handled by generator, am I right ?

**Comment by spalger** [comment]
Yeah, all you need to do is update the kibana version in your package.json and it will be golden. We haven't quite worked out the best way to generate plugins for specific versions of kibana...

**Comment by w33ble** [comment]
Related to https://github.com/elastic/generator-kibana-plugin/issues/50

~~This is fixed in [the template](https://github.com/elastic/template-kibana-plugin) by using a text input.~~ Nope, I misread the issue.

This actually seems like something that would be useful to add to the plugin-helpers, something that would bump the kibana version in the package.json automatically.
